### PR TITLE
Components: move WordPress Wordmark to package

### PR DESCRIPTION
**Pull request by arcangelini**
## Proposed Changes

This moves the WordPress Wordmark (the logo with the text WordPress next to it) into a packages/components to be consumed by other packages as well as Calypso.

Markup 2023-02-21 at 11 32 34

Markup 2023-02-21 at 11 33 04

## Testing Instructions

1. Pull branch and `yarn start`
2. Take a look at the areas where the logo is used.
3. http://calypso.localhost:3000/checkout/
4. http://calypso.localhost:3000/plugins ( WHILE LOGGED OUT )
5. While installing a plugin in the header

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/components/src/wordpress-watermark/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+} ) => {
+	return (
+		<svg
+			className={ className }
+			width={ size.width }
+			height={ size?.height }
+			viewBox="0 0 1007.7 172.4"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<g fill="#FFF">
```
Comment (alshakero) [inline packages/components/src/wordpress-watermark/index.tsx]: This should be configurable and default to `#fff`. Currently, we're override this value [with CSS](https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/universal-navbar-header/nav-style.scss#L99-L101). 

You'd only need to pass it [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/universal-navbar-header/index.jsx#L47) and the rest can default to white.

**Review by alshakero — COMMENTED** [review]
Beautiful. Tested and works great. Asked for a small improvement.

**Review by escapemanuele — APPROVED** [review]
I think this is great. Long life to packages 👌

**Review by tyxla — COMMENTED** [review]
Thanks @arcangelini 🙌

**Pull request merged**